src/Groups/perm.jl

```diff
@@ -639,6 +639,41 @@ function cycle_structures(G::PermGroup)
   return Set(cycle_structure(x) for x in r)
 end
 
+@doc raw"""
+    cycles(g::PermGroupElem)
+
+Return the cycles of the permutation `g` as a list of integer vectors.
```

Comment:
Does it return *all* cycles or only non-trivial ones? Maybe would be a good idea to pick an example with a trivial cycle (i.e. a fixed point) to show case that.

Reply:
It returns all cycles, I adjusted the example. Should one also write something about this in the doc string?

Reply:
For me it is fine if the example shows it. But I've noticed that since the convention is for hand-written representation to skip trivial cycles, students tend to forget about them. Perhaps this extends to more people and they'd appreciate the hint? 

Of course it might also cause confusion for other people ("why do they emphasize this? Am I missing something")... Dunno
```suggestion
Return all cycles (including trivial ones) of the permutation `g` as a list of integer vectors.
```

Independent of this, one might also wish to add some guarantees to the docstring:
```suggestion
Return all cycles (including trivial ones) of the permutation `g` as
a sorted list of integer vectors.
```